tests/MongoDB.Driver.Core.Tests/Core/Clusters/ClusterTests.cs

```diff
@@ -37,8 +37,8 @@ namespace MongoDB.Driver.Core.Clusters
 {
     public class ClusterTests
     {
-        private EventCapturer _capturedEvents;
-        private Mock<IClusterableServerFactory> _mockServerFactory;
+        private readonly EventCapturer _capturedEvents;
+        private readonly Mock<IClusterableServerFactory> _mockServerFactory;
```

Comment:
On line 47 below remove the `postServerSelector`.

Reply:
Done.